### PR TITLE
Add Pulsar Community Weekly Update - 2020-09-05~ 2020-09-11

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-09/2020-09-11-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+- [C++ Client] Support key-based batching for the C++ client.
+
+    https://github.com/apache/pulsar/pull/7996 (@[BewareMyPower](https://github.com/BewareMyPower))
+
+- [Pulsar Perf] Support setting the message key for consumers in the key-shared subscription mode.
+
+    https://github.com/apache/pulsar/pull/7989 (@[315157973](https://github.com/315157973))
+
+- [OAuth2] Get the Oauth2 token endpoint from the well-known configuration.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-11-pulsar-weekly.md:43]: Keep "OAuth2" format consistent.

File: content/2020/2020-09/2020-09-11-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+- [Bookie] Add `bookkeeperClientMinNumRacksPerWriteQuorum` and `bookkeeperClientEnforceMinNumRacksPerWriteQuorum` configuration options.
+
+    https://github.com/apache/pulsar/pull/7977 (@[mkozioro](https://github.com/mkozioro))
+
+- [Topic Policy] Support setting, getting, and removing `maxConsumers` on the topic level.
+
+    https://github.com/apache/pulsar/pull/7968 (@[zhanghaou](https://github.com/zhanghaou))
+
+- [Transaction] Update the transaction acknowledgement abort process on subscription.
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-11-pulsar-weekly.md:27]: refine the PR title.

File: content/2020/2020-09/2020-09-11-pulsar-weekly.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+- [Event]
+
+  -  Pulsar Summit Asia 2020
+
+     https://pulsar-summit.org/en/event/asia-2020
+
+  - 09/09: "Apache Pulsar: Under the Hood for Java Developers" by Addison Higham in the Israeli Java Community (Java.IL)
+
+    - Slides: https://docs.google.com/presentation/d/1zJdeFK9VVTDKL76UQimGWoJQOBlpEo5fmkGLbLh70GY/edit#slide=id.g944de4dcda_0_334
```
Comment (Jennifer88huang) [inline content/2020/2020-09/2020-09-11-pulsar-weekly.md:95]: shall we upload it to slideshare and then share the link?

**Comment by Huanli-Meng** [comment]
@Jennifer88huang comments are updated. Please help double check. Thanks

**Review by Jennifer88huang — APPROVED** [review]

**Comment by Jennifer88huang** [comment]
@Huanli-Meng great job, thanks.

**Pull request merged**